salt/tests/unit/modules/files/test_metalk8s_kubernetes_utils.yaml

```diff
@@ -278,3 +278,112 @@ get_service_endpoints:
       subsets: null
     raises: True
     result: "Unable to get kubernetes endpoints for my_service in namespace my_namespace"
+
+get_service_ips_and_ports:
+  # Multiples IPs and multiple ports
+  - obj:
+      spec:
+        clusterIPs:
+          - 10.0.0.1
+          - 10.0.0.2
+        clusterIP: 10.0.0.1
+        ports:
+          - name: http-web
+            port: 1234
+          - name: reloader-web
+            port: 1235
+    result:
+      ips:
+        - 10.0.0.1
+        - 10.0.0.2
+      ports:
+        http-web: 1234
+        reloader-web: 1235
```

Comment:
Can we add a test where `clusterIP` is different from first IP in `clusterIPs` just to check that `clusterIP` is always the first IP in result

Reply:
Done.